components/omega/doc/design/PGrad.md

```diff
@@ -0,0 +1,196 @@
+(omega-design-pressure-grad)=
+# Pressure Gradient
+
+
+## 1 Overview
+The pressure gradient will be responsible for computing the horizontal gradients of both the pressure and geopotential terms for the non-Boussinesq primative equations implemented in Omega.
+In the non-Boussinesq model, the vertical coordinate will be pressure as opposed to height.
+In a pure pressure coordinate the pressure gradient term disappears (since the pressure does not vary along lines of constant pressure), just as how the geopotential term disappears in a pure z coordinate model.
+However, similar to MPAS-Ocean's support for tilted height coordinates, Omega will allow for tilted pressure coordinates.
+This means that Omega will need to compute both the pressure and geopotential gradients.
```

Comment:
I think in several places it says the pressure gradient will compute the geopotential gradient as well, but then I don't see that as an output in the Design section below... am I missing something?

Reply:
I had `Geopotential` as an input to the `computePressureGrad` method with the thought it would compute both gradients and add them to `Tend` (MPAS-equivalent of `pressure_and_zmid`). I can certainly break this up into separate functions if that would be preferable.

Reply:
I added a line in Section 4.2.4 making it more explicit that `computePressureGrad` accumulates the sum of the pressure and geopotential gradients in `Tend`